code-challenges.test.js

```diff
@@ -14,46 +14,213 @@
+  // 6. Since all we have now is uppercased first letters of the words in the array. We need to add those capitalized letters to the rest of the word. We can achieve this by using the substring method and passing in the number 1 to get everything after the 0 index.
+  // 7. Add them both together in the return statment.
+  // 8. Run yarn jest to hopefully see tests passed.
+
+  const capitalized = (array) => {
+    return array.map((value) => {
+        return value[0].toUpperCase() + value.substring(1)
+    })
+  }
+
+  // PASS
+  // capitalized
+  //   ✓ returns an array with all the words capitalized
+  
+  // Test Suites: 1 passed, 1 total
+  // Tests:       2 passed, 2 total
+
+
+
 // --------------------3) Create a function that takes in a string and logs the index of the first vowel.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const vowelTester1 = "learn"
-// Expected output: 1
-const vowelTester2 = "academy"
-// Expected output: 0
-const vowelTester3 = "challenges"
-// Expected output: 2
+// const vowelTester1 = "learn"
+// // Expected output: 1
+// const vowelTester2 = "academy"
+// // Expected output: 0
+// const vowelTester3 = "challenges"
+// // Expected output: 2
+
+
+describe("firstVowel", () => {
+  it("logs the index of the first vowel", () => {
+    const vowelTester1 = "learn"
+    const vowelTester1Output = 1
+    const vowelTester2 = "academy"
+    const vowelTester2Output = 0
+    const vowelTester3 = "challenges"
+    const vowelTester3Output = 2
+    expect(firstVowel(vowelTester1)).toEqual(vowelTester1Output)
+    expect(firstVowel(vowelTester2)).toEqual(vowelTester2Output)
+    expect(firstVowel(vowelTester3)).toEqual(vowelTester3Output)
+  })
+})
+
+// ReferenceError: firstVowel is not defined
+
+// Test Suites: 1 failed, 1 total
+// Tests:       1 failed, 2 passed, 3 total
+
 
 // b) Create the function that makes the test pass.
+
+// Input: A string. More specifically, a word/set of characters consisting of both consonants and vowels.
+// Output: The index of the first occurance of a vowel in the string passed.
+
+// 1. Create a function named firstVowel.
+// 2. Pass in string as the parameter.
+// 3. Find a method whereby you match the characters of specific string.
+  // We can use the string method .match() to find a match of the vowels in the string.
+// 4. Declare a new variable equal to the parameter string. Specify with the .match() method what characters we want it to match with. In this case we will pass vowel characters.
+// 5. Find a method whereby you can find the index of a given characer.
+  // We can use the .indexOf() method to find the first index of those vowel characters by passing the new variable we made to store the matched vowel characters.
+// 6. Run yarn jest again to test code.
+
+const firstVowel = (string) => {
+  const firstOne = string.match('["a", "e", "i", "o", "u"]')
+  return string.indexOf(firstOne)
+}
```

Comment:
This looks excellent!  Oftentimes you will see people use the match method in conjunction with regex which is a really neat concept to dig into.  Here's some documentation if you are curious: [Regex](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions)